scss/_mixins.scss

```diff
@@ -0,0 +1,269 @@
+@mixin oTopperThemeElements {
+	.o-topper--has-headshot {
+		@include oTopperHasHeadshot;
```

Comment:
As a modifier this doesn't fit an element. It isn't quite a theme either as it isn't used standalone, but maybe would make most sense under `oTopperThemes` regardless?